build.properties

```diff
@@ -0,0 +1,5 @@
+VERSION=4.0.37
+RELEASE=1
+EPOCH=10
+SOURCE_BASE_URL=http://engci-maven.cisco.com/artifactory/STO-group/ciscosafec/4/0/37
```

Comment:
Internal Cisco, this should be devhub URL for external.  However, since the code is synced for non-public consumption, we will need to pull it in before build so its with the SOURCES dir (the rpmbuild step will not try and download again then).

Reply:
Did we just want to remove this line then and have it added to the container at the start?   That would mean removing some more from the spec file too.

Reply:
That's an option, we could remove the URL all together and have it such that SOURCE0 is just an expected item in the source directory.  I think this is a valid option

Reply:
Ok i'll just make sourceo the tar file name then.